cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -156,6 +156,19 @@
     private static final String ENV_VAR_HTTP_PROXY = "HTTP_PROXY";
     private static final String ENV_VAR_HTTPS_PROXY = "HTTPS_PROXY";
     private static final String ENV_VAR_NO_PROXY = "NO_PROXY";
+
+    protected static final String CO_ENV_VAR_CUSTOM_BRIDGE_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_BRIDGE_LABELS";
```

Comment:
It seems a bit weird that all these things are defined centrally here but used locally. Maybe this can be moved to the Models for different components?